### PR TITLE
DLPJTS-26 Add documentation to print sample BufferedImagePrintable object

### PR DESCRIPTION
**Pull request by datalogics-robl**
Document the purpose and use of the `BufferedImagePrintable` object, our local implementation of the `Printable` interface.

[DLPJTS-26](https://jira.datalogics.com/browse/DLPJTS-26)

### CODE REVIEW

**Comment by datalogics-devauto** [comment]
Build succeeded.

**Comment by datalogics-devauto** [comment]
Build succeeded.

File: src/main/java/com/datalogics/pdf/samples/printing/PrintPdf.java [hunk 1]
```diff
@@ -159,21 +156,52 @@ public static void printPdf(final URL inputUrl) throws Exception {
...
+    /**
+     * A Printable implementation that provides rasterized pages of the document.
+     *
+     * <p>
+     * This class interacts with the Java print architecture, providing a printable image of each page on demand. See
+     * the Java AWT {@link https://docs.oracle.com/javase/7/docs/api/java/awt/print/Printable.html Printable} interface
+     * for more information.
+     */
     private static class BufferedImagePrintable implements Printable {
+        private int previousPageIndex = -1;
```
Comment (jrowden8) [inline src/main/java/com/datalogics/pdf/samples/printing/PrintPdf.java:168]: These static variables are right at home now where they should be. :+1:

**Comment by jrowden8** [comment]
The documentation looks good!

**Pull request merged**